setup.py

```diff
@@ -50,13 +50,15 @@ def parse_arg_remove_string(argv, arg_name_equal):
 
 cuda_version = None
 rocm_version = None
+is_rocm = False
 # The following arguments are mutually exclusive
 if parse_arg_remove_boolean(sys.argv, '--use_tensorrt'):
     package_name = 'onnxruntime-gpu-tensorrt' if not nightly_build else 'ort-trt-nightly'
 elif wheel_name_suffix == 'gpu':
     # TODO: how to support multiple CUDA versions?
     cuda_version = parse_arg_remove_string(sys.argv, '--cuda_version=')
 elif parse_arg_remove_boolean(sys.argv, '--use_rocm'):
+    is_rocm = True
```

Comment:
How about windows build? Should we do additional check (when not Windows) to toggle it to `True`?

Reply:
For Windows, --use_rocm should not be used.